sv_mini_atlas/tests/test_urn.py

```diff
@@ -28,4 +30,15 @@ def test_urn_invalid_component():
     key = 5
     with pytest.raises(ValueError) as excinfo:
         URN(urn).up_to(key)
-        assert str(excinfo.value) == "URN has no component: exemplar"
+        assert str(excinfo.value) == "URN has no componentemplar"
```

Comment:
did something get accidentally deleted here?

Reply:
Huh! I guess so... That's weird it's passing but i've noticed some weirdness with pytest `excinfo.value` before passing when it shouldn't. Fixing.